site/docs/guides/tls.md

```diff
@@ -0,0 +1,83 @@
+# TLS Ingress with Minikube
+
+In this guide we walk through the process of configuring a Nessie Service with secure HTTPS transport in
+[Minikube](https://minikube.sigs.k8s.io/docs/).
+
+## Setting up Minikube
+
+1. Start Minikube cluster: `minikube start`
+2. Enable NGINX Ingress controller: `minikube addons enable ingress`
+
+## Creating Certificates
+
+If you have your own trusted certificate you can use it to sign a new certificate for the Nessie service.
+
+Otherwise, the rest of this section shows how can create a self-signed certificate and add it the list of trusted
+certificates in the client OS.
+
+To generate a fresh pair of a key and a (self-signed) certificate:
+
+```shell
+openssl req -new -subj "/CN=Nessie" -addext "subjectAltName = DNS:nessie.local" \
+  -newkey rsa:2048 -keyout nessie-key.pem -out nessie.crt -noenc
+```
+
+Note the `-noenc` option. It is used only for the sake of simplicity of this example deployment. 
```

Comment:
I see this option [here](https://www.openssl.org/docs/man3.1/man1/openssl-req.html), but on macOS, although I do have openssl 3.1.0, the option is not available:

```
openssl req -new -subj "/CN=Nessie" -addext "subjectAltName = DNS:nessie.local" \
  -newkey rsa:2048 -keyout nessie-key.pem -out nessie.crt -noenc
req: Unrecognized flag noenc
req: Use -help for summary.
```

It seems we must use `-nodes` instead. Maybe we should remove this option, if it's not strictly required for the tutorial?

Reply:
fixed